design/app-to-app.md

```diff
@@ -0,0 +1,80 @@
+App to App Interactions
+===========
+
+This document describes an extension to the SDK to allow MRE apps to receive content (e.g., photos) from the MRE clients. Content may be generated by the client itself, or, in future iterations not yet described here, may arrive from other apps with coordination from a particular client.
+
+API Design
+-----------
+This design introduces the concept of a `SharingMessage`, a base class of content-specific messages, such as `PhotoSharingMessage`. We provide an outline for the implementation of `PhotoSharingMessage` only, but expect to add more sharing capabilities over time.
+
+`SharingMessage` is itself relatively straightforward, but contains enough information to allow the recipient (callback listener) to cast it to the appropriate type (e.g., `PhotoSharingMessage`).
```

Comment:
Maybe add something here about why: we envision multiple message types (and perhaps related types) and don't want to add a registration api for each

Reply:
I agree. Ensuring coverage of who, what, why provides a lot of clarity.